WordPress/WordPressTest/BlogDetailsViewController+MeButtonTests.swift

```diff
@@ -58,40 +62,4 @@ class BlogDetailsViewControllerTests: XCTestCase {
         XCTAssertEqual(meButton.accessibilityLabel, TestConstants.meButtonLabel)
         XCTAssertEqual(meButton.accessibilityHint, TestConstants.meButtonHint)
     }
-
-    func testPresentMeOnButtonTap() {
```

Comment:
I know removing a unit test isn't great, but an explanation on this:

1. I really tried to write a test that made sense here but failed.  I spent around 2 hrs today trying to figure this out, because...
2. ...there's no good support in UIKit to programmatically tap on a view or trigger a gesture recognizer.  In fact because of this I also noticed that...
3. ...the original test isn't really testing a tap either.  It's calling `#selector(BlogDetailsViewController.presentHandler)` in a kind of indirect way.

I can still drop the PR but I feel it makes the code a bit more flexible in case we want to show the Me scene from a new view controller, and avoid code duplication.

Reply:
Agreed. Testing a tap is more a matter of UI testing. I think it's fine to remove this.

Reply:
Thanks, @Gio2018 !